docs/devGuide/projectManagement.md

```diff
@@ -71,81 +71,47 @@ For general best practices, refer to the guide [_Working with PRs_ @SE-EDU](http
+   Do take a peek at the diff for `components.min.js` to see if there's any strange changes (e.g. the version of jQuery changes even though no one upgraded it).
    </box>
 
-1. **Commit** the new changes (`git commit -am 'Update dist'`)
-
-1. **Tag** the new commit with the new version number, whereby `XYZ` is the incremented number of the previous release.<br>
-   e.g., `git tag v2.0.1-markbind.XYZ`
-
-   <box type="important" seamless>
-
-   The tag used is a lightweight tag. _Don't_ use an annotated tag.
-   </box>
-
-1. **Push** everything to the main vue-strap repository (replace `XYZ` with version number).
-
-   ```
-   $ git push upstream master
-   $ git push upstream v2.0.1-markbind.XYZ
-   ```
-
 ### Building MarkBind
 
-1. **Navigate to the root** directory for MarkBind
-1. **Update dependencies** by running `npm install`
-1. **If we did a new release for the Vue components**:
-
-   Copy `vue-strap.min.js` from the vue-strap repository to the main asset folder, and to each of the test site's `expected/` folder.
-
-   ```sh
-   # copy to main asset folder
-   $ cp frontend/components/dist/vue-strap.min.js asset/js/vue-strap.min.js
-   ```
-   
-   Run `npm updatetest` and ensure only the minified bundle is copied to each of the expected test site's folders. If there are other changes, discard them for now - refer to **Rebuild the test files**.
-
-   Commit only the new vue-strap assets (replace `XYZ` with version number).
-
-   ```sh
-   $ git commit -m 'Update vue-strap version to v2.0.1-markbind.XYZ'
-   ```
-
-   <box type="info" seamless>
-
-   We uniquely do this for each MarkBind release (rather than spontaneously update the vue-strap files for each affected PR), in order to reduce unnecessary merge conflicts. It also makes it easier for the maintainers to vet the changes.
-   </box>
+1. **Make sure to start with a "clean slate"** by running `npm run ci:all` or `npm run ciwin:all` if you haven't done so previously.
 
 1. **Increment the version number** by running `npm version`. Which to increment (`patch`, `minor` or `major`) depends on what PRs are merged for the new version, which means you must know beforehand about the changes.
 
    * If there are no significant changes, a `patch` is sufficient: `npm version patch`
    * If there are significant changes (e.g. breaking changes, new release), a `minor` release is needed:
      1. Run `npm version minor` as per normal.
-     1. Update the version number inside `src/lib/markbind/package-lock.json` and `src/lib/markbind/package.json` manually. This is because `npm version` will not automatically update the numbers from the outside. (Note: You do not have to commit these changes immediately, as we also have to rebuild the test files in the next step anyway.)
+        
+     1. Update the version number inside the `frontend/components` and `src/lib/markbind` packages similarly, as `npm version` does not update the subpackage versions.
```

Comment:
Hmm I mean to use `npm version` for the subpackages as well, since it dosen't create an extra commit there and updates both `package/package-lock.json`

Reply:
Oh, that wasn't clear.

Doesn't create an extra commit where?

Reply:
in the root repo (the commit autotagged with the version number)

reworded it slightly

Reply:
I don't understand. Isn't there only one repo now?

Reply:
yup, so only the first `npm version` would create the autotagged commit, the subsequent ones automatically update the subpackages' `package/package-lock.json` version numbers

Reply:
Ok, the revised sentence is clearer.

> 1. Update the version number inside the `frontend/components` and `src/lib/markbind` packages similarly using `npm version minor` (as `npm version` does not update the subpackage versions automatically).